app/views/learning_hours_mailer/learning_hours_report_email.html.erb

```diff
@@ -0,0 +1,80 @@
+    <tbody style="box-sizing: border-box;">
+      <tr class="row" style="box-sizing: border-box; vertical-align: top;" valign="top">
+        <td class="main-body-cell" style="box-sizing: border-box;">
+          <table class="container" style="box-sizing: border-box; font-family: Helvetica, serif; min-height: 150px; padding-top: 5px; padding-right: 5px; padding-bottom: 5px; padding-left: 5px; margin-top: auto; margin-right: auto; margin-bottom: auto; margin-left: auto; height: 0px; width: 90%; max-width: 550px;" width="90%" height="0">
+            <tbody style="box-sizing: border-box;">
+              <tr style="box-sizing: border-box;">
+                <td class="container-cell" style="box-sizing: border-box; vertical-align: top; font-size: medium; padding-bottom: 50px;" valign="top">
+                  <table class="c1766" style="box-sizing: border-box; margin-top: 0px; margin-right: auto; background:#728089;margin-left: 0px; padding-top: 5px; padding-right: 5px; padding-bottom: 5px; padding-left: 5px; width: 100%; min-height: 30px; background:#fff; min-height: 150px; margin-bottom: 20px; border-top-left-radius: 3px; border-top-right-radius: 3px; border-bottom-right-radius: 3px; border-bottom-left-radius: 3px;" width="100%" height="0">
+                    <tbody style="box-sizing: border-box;">
+                      <tr style="box-sizing: border-box;">
+                        <td class="cell c1776" style="box-sizing: border-box; width: 70%; vertical-align: middle;" width="70%" valign="middle">
+                          <div class="c1144" style="box-sizing: border-box; padding-top: 10px; padding-right: 10px; padding-bottom: 10px; padding-left: 10px; font-size: 17px; font-weight: 500;text-align: center;color: #fff;">Court
+                            Appointed Special Advocate (CASA) / <%= @casa_organization&.name %>
+                            <br style="box-sizing: border-box;">
+                          </div>
+                        </td>
+                      </tr>
+                      <tr style="box-sizing: border-box;">
+                        <td class="card-content" style="box-sizing: border-box; font-size: 13px; line-height: 20px; color: rgb(111, 119, 125); padding-top: 10px; padding-right: 20px; padding-bottom: 0px; padding-left: 20px; vertical-align: top;" valign="top">
+                          <h3 style="box-sizing: border-box; font-weight: 800; margin: 20px 0 5px; font-size: 16px;">Learning hours have been added this week!</h3>
+                          <p style="box-sizing: border-box;">Please see the attached full report or export the full report at <a href="https://casa-qa.herokuapp.com/reports" style="box-sizing: border-box;">https://casa-qa.herokuapp.com/reports</a>.</p>
```

Comment:
We should use routes here so we don't end up with the QA site hardcoded -- `link_to reports_url reports_url`

Reply:
got it!